### ISSUE TITLE
Training detail & loss

### ISSUE THREAD

**Issue by shawnsya** [MouseLand/cellpose#473]
During training from scratch,i found non-convergence issue.
Dataset provided by your official [website:](https://www.cellpose.org/dataset)
Training settings follows your doc.
`python -m cellpose --train --train_size --use_gpu --dir ~/cellpose_dataset/train/ --test_ ˓→dir ~/cellpose_dataset/test/ --img_filter _img --pretrained_model None --chan 2 -- ˓→chan2 1`
I want to know how loss declined and your lr schedule when you trained cellpose model to see where is my problem..
<img width="326" alt="image" src="https://user-images.githubusercontent.com/32290891/158302807-58cd7ac1-efd5-44ce-9a63-b7f83202f06d.png">

**Comment by kevinjohncutler** [comment]
@shawnsya looks to me like the loss is converging. I do think having a built-in tracker for loss and all the hyperparameters and scheduling would be useful. Maybe you could implement this and make a pull request? Possibly weights and biases or another package could be utilized and turned on with a CLI flag. That said, there are already logs (not sure where these are save to) that store much of this info, just not as convenient to parse. As to general debugging of training, I've always just added in some print and save commands to figure out what is going awry.

**Comment by shawnsya** [comment]
@kevinjohncutler Thanks! It turns out that the loss is converging... Thanks for your patience and experience anyway:)
